src/platforms/common/blackpill-f4/README.md

```diff
@@ -26,39 +31,45 @@ This directory contains the common code for the following platforms:
 | LED UART        | PA4    | PA1                  | LED       |
 | User button KEY | PA0    | PA0                  |           |
 
-## Instructions for Makefile buildsystem and ST MaskROM
-### How to Build
+## Instructions for build system
 
-In the following commands replace `blackpill-f4x1cx` with the platform you are using, e.g. `blackpill-f401cc`.
+### Variant with meson and BMD bootloader enabled (default)
 
-To build the code using the default pinout, run:
+0. Clone the repo and libopencm3 submodule, install toolchains, meson, etc.
 
 ```sh
+git clone https://github.com/blackmagic-debug/blackmagic.git
 cd blackmagic
-make clean
-make PROBE_HOST=blackpill-f4x1cx
+git submodule update --init
 ```
 
-or, to use alternative pinout 1, run:
+1. Create a build configuration for specific platform (WeActStudio F401CC, F401CE, or F411CE) with specific options (enable BMD bootloader flags and offsets -- this is the default)
 
 ```sh
-cd blackmagic
-make clean
-make PROBE_HOST=blackpill-f4x1cx ALTERNATIVE_PINOUT=1
+meson setup build_blackpill_xxxxxx --cross-file=cross-file/blackpill-xxxxxx.ini -Dbmd_bootloader=true
```

Comment:
It's perhaps pertinent to default this to `build` as the build directory name but include a line that points out that this name is arbitrary and that you can set multiple build configurations up in parallel by varying this name?

Reply:
All changed to simply 'build'